modules/tensor_mechanics/src/base/TensorMechanicsApp.C

```diff
@@ -234,6 +237,9 @@ TensorMechanicsApp::registerObjects(Factory & factory)
   registerAux(NewmarkVelAux);
 
   registerBoundaryCondition(Pressure);
+
+  registerPostprocessor(Mass);
```

Comment:
Some of the devs have actually wanted to go away from more macros to where everything is simply `registerObject()`. I can see ups and downs. We should probably be consistent though but Stephanie doesn't need to do worry about it with this PR.